### PR TITLE
New documentation

### PR DESCRIPTION
**Pull request by JxxIT**
Adds documentation for some endpoints

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1335,6 +1338,306 @@ You can either provide a date range or a single date. If you provide a single da
...
+            "eindDatum": "2023-10-20T00:00:00.000+02:00"
+        },
+        ...
+    ]
+}
+```
+
+</details>
+
+### Studiemateriaal: `GET /rest/v1/vakanties/leerling/[id]` & `GET /rest/v1/vakanties/leerling/[id]/vak[uuid]`
```
Comment (MattterSteege) [inline README.md:1425]: I do not think that the urls in this portion of the docs are correct. 

If i'm correct the urls should be:

`/rest/v1/vakken/studiemateriaal/[leerling id]` for all subjects with studiemateriaal
`/rest/v1/studiemateriaal/algemeen/[leerling id]` for the general studiemateriaal (I think that the links to online books are here, but I'm not sure.
`/rest/v1/studiemateriaal/[leerling id]/vak/[vak id]` for the jaarbijlagen, lesstof etc.

so go and check that out :)

File: README.md [hunk 2]
```diff
@@ -1335,6 +1338,306 @@ You can either provide a date range or a single date. If you provide a single da
...
+### Vakanties: `GET /rest/v1/vakanties/leerling/[id]`
+<details><summary>Click to open</summary>
+
+Fetches info about the school, including your mentor.
+
+#### Parameters
+
+| Name                       | Type      | Value                 |
+|----------------------------|-----------|-----------------------|
+| Authorization              | Header    | Bearer [access_token] |
```
Comment (MattterSteege) [inline README.md:1384]:
```suggestion
| id            | URL    | [user id]             |
| Authorization | Header | Bearer [access_token] |
```

File: README.md [hunk 3]
```diff
@@ -1335,6 +1338,306 @@ You can either provide a date range or a single date. If you provide a single da
...
+### Schoolgegevens: `GET /rest/v1/[id]/schoolgegevens`
+<details><summary>Click to open</summary>
+
+Fetches info about the school, including your mentor.
+
+#### Parameters
+
+| Name                       | Type      | Value                 |
+|----------------------------|-----------|-----------------------|
+| Authorization              | Header    | Bearer [access_token] |
```
Comment (MattterSteege) [inline README.md:1352]:
```suggestion
| id            | URL    | [user id]             |
| Authorization | Header | Bearer [access_token] |
```

File: README.md [hunk 4]
```diff
@@ -1335,6 +1338,306 @@ You can either provide a date range or a single date. If you provide a single da
...
+
+Fetches all studiemateriaal.
+
+First, make a request to `GET /rest/v1/vakanties/leerling/[id]`. And then to `GET /rest/v1/vakanties/leerling/[id]/vak[uuid]` with the UUID. 
+
+#### Parameters
+
+| Name                       | Type      | Value                 |
+|----------------------------|-----------|-----------------------|
+| Authorization              | Header    | Bearer [access_token] |
```
Comment (MattterSteege) [inline README.md:1436]:
```suggestion
| id            | URL    | [user id]             |
| Authorization | Header | Bearer [access_token] |
```

**Review by MattterSteege — CHANGES_REQUESTED** [review]
okay, really nice too. The studiemateriaal is a bit messy, but over all the addition to the docs are nice.

**Comment by JxxIT** [comment]
@MattterSteege Jeez, how did I miss that?! Fixed now.

File: README.md [hunk 5]
```diff
@@ -30,6 +30,9 @@
     - [School Years: `GET /rest/v1/schooljaren`](#schooljaren-get-restv1schooljaren--get-restv1schooljarenid)
     - [Vakkeuzes: `GET /rest/v1/vakkeuzes`](#vakkeuzes-get-restv1vakkeuzes)
     - [Waarnemingen: `GET /rest/v1/waarnemingen`](#waarnemingen-get-restv1waarnemingen)
+    - [Schoolgegevens: `GET /rest/v1/leerlingen/[id]/schoolgegevens`](#schoolgegevens-get-restv1idschoolgegevens)
+    - [Vakanties: `GET /rest/v1/vakanties/leerling/[id]`](#vakanties-get-restv1vakantiesleerlingid)
+    - [Studiemateriaal: `GET /rest/v1/vakken/studiemateriaal/[id]` & `GET /rest/v1/vakken/studiemateriaal/[id]/vak[uuid]`](#studiemateriaal-get-restv1vakantiesleerlingid--get-restv1vakantiesleerlingidvakuuid)
```
Comment (MattterSteege) [inline README.md:35]:
```suggestion
    - [Studiemateriaal: `GET /rest/v1/vakken/studiemateriaal/[id]` & `GET /rest/v1/vakken/studiemateriaal/[id]/vak/[uuid]`](#studiemateriaal-get-restv1vakantiesleerlingid--get-restv1vakantiesleerlingidvakuuid)
```

File: README.md [hunk 6]
```diff
@@ -1335,6 +1338,309 @@ You can either provide a date range or a single date. If you provide a single da
...
+            "eindDatum": "2023-10-20T00:00:00.000+02:00"
+        },
+        ...
+    ]
+}
+```
+
+</details>
+
+### Studiemateriaal: `GET /rest/v1/vakken/studiemateriaal/[id]` & `GET rest/v1/vakken/studiemateriaal/[id]/vak[uuid]`
```
Comment (MattterSteege) [inline README.md:1427]:
```suggestion
### Studiemateriaal: `GET /rest/v1/vakken/studiemateriaal/[id]` & `GET rest/v1/vakken/studiemateriaal/[id]/vak/[uuid]`
```

File: README.md [hunk 7]
```diff
@@ -1335,6 +1338,309 @@ You can either provide a date range or a single date. If you provide a single da
...
+<details><summary>Click to open</summary>
+
+Fetches all studiemateriaal.
+
+First, make a request to `GET /rest/v1/vakken/studiemateriaal/[id]`. And then to `/rest/v1/vakken/studiemateriaal/[id]/vak[uuid]` with the UUID.
```
Comment (MattterSteege) [inline README.md:1432]:
```suggestion
First, make a request to `GET /rest/v1/vakken/studiemateriaal/[id]`. And then to `/rest/v1/vakken/studiemateriaal/[id]/vak/[uuid]` with the UUID. 
```

File: README.md [hunk 8]
```diff
@@ -1335,6 +1338,309 @@ You can either provide a date range or a single date. If you provide a single da
...
+            "additionalObjects": {},
+            "afkorting": "ne",
+            "naam": "Nederlandse taal",
+            "UUID": "REDACTED"
+        },
+        ...
+    ]
+}
+```
+`GET /rest/v1/vakken/studiemateriaal/[id]/vak[uuid]` returns:
```
Comment (MattterSteege) [inline README.md:1479]:
```suggestion
`GET /rest/v1/vakken/studiemateriaal/[id]/vak/[uuid]` returns:
```

**Review by MattterSteege — CHANGES_REQUESTED** [review]
Saw this a bit too late too... but after this I think it's ready

**Comment by JxxIT** [comment]
@MattterSteege Sorry for my stupidness!

**Comment by MattterSteege** [comment]
ahh, it's okay, looks really nice tho👍

**Comment by JxxIT** [comment]
> ahh, it's okay, looks really nice tho👍

Thanks!

**Pull request merged**